frontend/src/components/MultiSelect.js

```diff
@@ -158,42 +160,60 @@ function MultiSelect({
     }
   };
 
+  const onKeyDown = (e) => {
+    if (e.key === 'Enter' || e.key === ' ') {
+      e.preventDefault();
+      selectorRef.current.focus();
+      onClick();
+    }
+  };
+
   const Selector = canCreate ? Creatable : Select;
 
   return (
     <Controller
       render={({ onChange: controllerOnChange, value, onBlur }) => {
         const values = value ? getValues(value) : value;
         return (
-          <Selector
-            className="ttahub-multi-select margin-top-1"
-            id={name}
-            value={values}
-            onBlur={onBlur}
-            onChange={(event) => {
-              if (onItemSelected) {
-                onItemSelected(event);
-              } else if (event) {
-                onChange(event, controllerOnChange);
-              } else {
-                controllerOnChange([]);
-              }
-            }}
-            inputId={inputId}
-            styles={styles(singleRowInput)}
-            components={{ ...componentReplacements, DropdownIndicator }}
-            options={options}
-            isDisabled={disabled}
-            tabSelectsValue={false}
-            isClearable={multiSelectOptions.isClearable}
-            closeMenuOnSelect={multiSelectOptions.closeMenuOnSelect || false}
-            controlShouldRenderValue={multiSelectOptions.controlShouldRenderValue}
-            hideSelectedOptions={multiSelectOptions.hideSelectedOptions}
-            placeholder={placeholderText || ''}
-            onCreateOption={onCreateOption}
-            isMulti
-            required={!!(required)}
-          />
+          // eslint-disable-next-line jsx-a11y/no-static-element-interactions
+          <div
+            onClick={onClick}
+            onKeyDown={onKeyDown}
+            data-testid={`${name}-click-container`}
+            // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+            tabIndex={disabled ? 0 : undefined}
+          >
```

Comment:
I think we need another div here in between the div with the on click handler and the selector, conditionally aria-hiding the select. Otherwise we get weirdness where the placeholder is read without context.

For example:

```jsx
        <div
            onClick={onClick}
            onKeyDown={onKeyDown}
            data-testid={`${name}-click-container`}
            // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
            tabIndex={disabled ? 0 : undefined}
          >
            <div aria-hidden={disabled}>
              <Selector

Reply:
got it: 11ad181b163c6c84ad1fb8248ae2d8be61531fee